doc/development/how-to-release.md

```diff
@@ -4,15 +4,15 @@ The core release process is automated with github [actions](https://github.com/r
   - Select the base branch that you would like to make a release from (usually this will be "master") in the drop down.
   - Enter the version of the new release (eg. "10.123.1" or "4.5.0-CustDemo")
 
-2. This will create a PR, which you should look over and get someone else on the team to review. Verify the changelog by checking the PR description which has a list of commits since the last tag was made. If any changes are required, commit them to this PR's branch.
+2. This will create a PR, which you should look over and get someone else on the team to review. Verify the changelog by checking the PR description which has a list of commits since the last tag was made. If any changes are required, commit them to this PR's branch (please reorder commits such that the version change comes last as the last commit will be the one tagged). Verify that the base commit on the branch you are releasing has passed evergreen checks (or that the failures are known to be spurious and are tracked).
 
-3. When ready, merge the PR. You can squash if there are only "prepare release" changes, but use a "merge-commit" strategy if there are functional changes added manually to the PR (note that if using a merge strategy, the last commit after merge will be the one tagged, so you may want to reorder the commits so that the 'prepare' commit comes last). On merge, the "make-release" action will run. This will:
+3. When ready, merge the PR. You can squash if there are only "prepare release" changes, but use a "merge-commit" strategy if there are functional changes added manually to the PR. On merge, the "make-release" action will run which triggers a Github deployment. Someone from the `@realm/realm-core-team` must approve the deployment for this step to run. Find the Github [deployment](https://github.com/realm/realm-core/actions/workflows/make-release.yml) and approve it. This will:
```

Comment:
Can this be approved by the person who submitted the deployment, or does it have to be someone else?

Reply:
Yes, it can be approved by the person who initiated the deployment. The security layer is there so that only approved contributors (@realm/realm-core-team) can make a deployment and not anyone outside the organization. This is because the deployment action gets access to environment variable "secrets", which right now only consists of the URL of the slack integration that posts to our release announcement channel. I think this layer isn't strictly necessary because our branch is protected to only allow members of our org to merge pull requests, (and that is how the GHA is triggered) but it is a good security practice to put in place now, so that we can build on it as the process evolves over time.